mix.exs

```diff
@@ -98,7 +98,7 @@ defmodule Trento.MixProject do
       {:timex, "~> 3.7.7"},
       {:trento_contracts,
        github: "trento-project/contracts",
-       ref: "b8e1036d0177c029ac40d34f237c366a7fb08bec",
+       ref: "53bce57d28b18fd4d73cab30a6e0793b8204139b",
```

Comment:
This must be updated once the contracts project PR is merged I guess

Reply:
@nelsonkopliku , at the end you didn't use the merge commit, you used an intermediate commit that it is not in `main` :see_no_evil: 
I think we should've put this instead: https://github.com/trento-project/contracts/commit/cd285f49e0cbdf168ae9aa3d86d92252b3754139

Reply:
right, my bad :sweat_smile: opening a pr to pin to that commit. pardon eheh

Reply:
@arbulu89 here you go https://github.com/trento-project/web/pull/1580